app/src/main/java/org/p2p/wallet/receive/eth/ReceiveEthereumFragment.kt

```diff
@@ -0,0 +1,70 @@
+package org.p2p.wallet.receive.eth
+
+import androidx.core.view.isVisible
+import android.graphics.Bitmap
+import android.os.Bundle
+import android.view.View
+import org.koin.android.ext.android.inject
+import org.p2p.core.glide.GlideManager
+import org.p2p.wallet.R
+import org.p2p.wallet.common.mvp.BaseMvpFragment
+import org.p2p.wallet.databinding.FragmentEthereumReceiveBinding
+import org.p2p.wallet.utils.args
+import org.p2p.wallet.utils.copyToClipBoard
+import org.p2p.wallet.utils.popBackStack
+import org.p2p.wallet.utils.viewbinding.viewBinding
+import org.p2p.wallet.utils.withArgs
+
+private const val ARG_TOKEN_LOGO_URL = "ARG_TOKEN_LOGO_URL"
+private const val ARG_TOKEN_SYMBOL = "ARG_TOKEN_SYMBOL"
+
+class EthereumReceiveFragment :
+    BaseMvpFragment<ReceiveEthereumContract.View,
```

Comment:
strange formatting, maybe you can you typealias to make it more readable
or static imports